packages/extension-dapp/src/compat/index.ts

```diff
@@ -1,11 +1,13 @@
 // Copyright 2019-2021 @polkadot/extension-dapp authors & contributors
 // SPDX-License-Identifier: Apache-2.0
 
+import initMetaMaskSource from './metaMaskSource';
 import singleSource from './singleSource';
 
 // initialize all the compatibility engines
 export default function initCompat (): Promise<boolean> {
   return Promise.all([
-    singleSource()
+    singleSource(),
```

Comment:
Note-to-self: After this can drop the singleSource. It was always great as a reference, but now that we have a proper reference available here, not needed anymore.